src/js/easymde.js

```diff
@@ -1762,6 +1762,11 @@ EasyMDE.prototype.markdown = function (text) {
 
         // Convert the markdown to HTML
         var htmlText = marked(text);
+        
+        // Sanitize HTML
+        if (this.options.renderingConfig && typeof this.options.renderingConfig.sanitizerFunction === 'function') {
+            htmlText = this.options.renderingConfig.sanitizerFunction(htmlText);
```

Comment:
I advice you to do : 

```js
    htmlText = this.options.renderingConfig.sanitizerFunction.call(this, htmlText);
```

Sometime is usefull to get `this` in event or callback like this.

Reply:
Thanks, changes pushed.